src/prompt.ts

```diff
@@ -153,69 +152,65 @@ async function runOrReviseFlow(
   apiEndpoint: string,
   silentMode?: boolean
 ) {
-  const nonEmptyScript = script.trim() !== '';
+  const EmptyScript = script.trim() === '';
```

Comment:
in this codebase we generally try to reserve PascalCase for classes and types
```suggestion
  const emptyScript = script.trim() === '';
```